modules/onMath/index.js

```diff
@@ -46,7 +59,33 @@ module.exports.msg = function(text, from, reply, raw) {
 
   var res = MathScopeEval(text);
 
-  if(res !== null) {
+  // If res is null or just echos our input, filter it
+  if(res === null || res == text) {
+    return;
+  }
+
+  res = res.toString();
+
+  // If the response is javascript
+  if(javascript.test(res)) {
+    return;
+  }
+
+  // Truncate our responses (reckless-irc-exec)
+  if(res.length < 400) {
     reply(res);
   }
+  else {
+    reply(res.substring(0,396) + " ...");
+    bot.sayTo(from, 'All of your command output: ' + res);
+  }
 };
+
+module.exports.command = "reset";
```

Comment:
minor, I think using the form `module.exports.run_reset = ...` is more readable here since this command is auxiliary to the purpose of the module.